source/plugins/projects/projects/index.js

```diff
@@ -777,7 +777,7 @@ let projects = {
 	 *
 	 * @example
 	 * 
-	 * newFolder('MyNewProject', 'C:\Users\User\Desktop');
+	 * newFolder(project, 'C:\Users\User\Desktop');
```

Comment:
The second parameter is a folder within the project, it cannot be with a full path, it has to start with /...